tests/test_mtm_condensed.py

```diff
@@ -121,10 +120,11 @@ def get_torsion(x_l):
         ubps, params, masses, coords, box, temperature, pressure, num_equil_steps, seed
     )
 
+    lamb = 0.0
     # test with both frozen masses and free masses
     for test_masses in [frozen_masses, masses]:
 
-        # (ytz): emprically scanning over multiple Ks seem to suggest 50 is a sweet spot
+        # (ytz): emprically scanning over multiple Ks seem to suggest 100 is a sweet spot
```

Comment:
Curiosity: Is this from a using a different scan approach?

Reply:
This was from talking to @proteneer about the inconsistency between the comment and the code. So no change besides correcting the comment